gto-support-recyclerview/src/main/java/org/ccci/gto/android/common/recyclerview/adapter/AbstractDataBindingAdapter.kt

```diff
@@ -12,17 +12,21 @@ abstract class AbstractDataBindingAdapter<B : ViewDataBinding, VH : DataBindingV
     private val lifecycleOwner by weak(lifecycleOwner)
 
     // region Lifecycle
-    final override fun onCreateViewHolder(parent: ViewGroup, viewType: Int) =
-        onCreateViewDataBinding(parent, viewType)
-            .also { it.lifecycleOwner = lifecycleOwner }
-            .let { onCreateViewHolder(it, viewType) }
+    final override fun onCreateViewHolder(parent: ViewGroup, viewType: Int) = onCreateViewHolder(
```

Comment:
I may be mistaken here so please let me know, but here we are overriding```AbstractDataBindingAdapter```'s ```onCreateViewHolder``` method in favor of our own logic of how this method should work(?). I see we override the ```onCreateViewHolder``` in favor of using another ```onCreateViewHolder```. Is the one we are setting it to referencing the one below? 
```kotlin
protected abstract fun onCreateViewHolder(binding: B, viewType: Int): VH
```
If so, does that ```onCreateViewHolder``` get defined within the GodTools app, as well as ```onViewDataBindingCreated```? If so could you give an example of where it would be set?

Reply:
let's jump on a call to talk through the design here and reason for these abstract adapters, it'll be easier than typing it all out